tools/auto-compose/tests/test_dagbuilder.py

```diff
@@ -0,0 +1,112 @@
+import os
+import datetime
+
+import pytest
+from airflow import DAG
+from airflow.operators.bash_operator import BashOperator
+
+from dagfactory import dagbuilder
+
+here = os.path.dirname(__file__)
+
+
+class TestDagBuilder(object):
```

Comment:
refactor to use [unittest](https://docs.python.org/3/library/unittest.html) framework.